apix-impl/61/overrides.gradle

```diff
@@ -5,5 +5,5 @@ ext {
     alfresco_repo_version = '7.33.12'
     alfresco_dm_version = alfresco_61_dm_version
     alfresco_min_version = "6.1"
-    //Not setting alfresco_max_version here to make it easier to test on next Alfresco version (only for latest version)
+    alfresco_max_version = "6.1.99"
```

Comment:
Property still needs to be added to `apix-impl/61/src/main/config/module.properties`?

Reply:
Good catch.